src/sentry/migrations/0375_modify_alertruletriggeraction_and_create_notificationaction.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 2.2.28 on 2023-03-08 00:50
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0374_breaking_even_more_keys"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="alertruletriggeraction",
+            name="target_type",
+            field=sentry.db.models.fields.bounded.BoundedIntegerField(),
+        ),
+        migrations.AlterField(
+            model_name="alertruletriggeraction",
+            name="type",
+            field=sentry.db.models.fields.bounded.BoundedIntegerField(),
+        ),
+        migrations.CreateModel(
+            name="NotificationAction",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("trigger_type", sentry.db.models.fields.bounded.BoundedIntegerField()),
```

Comment:
Should this model have an organization_id? It looks like `integration`, `project` and `sentry_app` are all nullable which makes this model problematic for cleanup operations that remove organizations or projects.